app/src/main/java/com/xmartlabs/doapp/MilisecondsLocalDateAdapter.java

```diff
@@ -0,0 +1,21 @@
+package com.xmartlabs.doapp;
+
+import com.raizlabs.android.dbflow.converter.TypeConverter;
+
+import org.threeten.bp.LocalDate;
+
+/**
+ * Created by scasas on 3/9/17.
+ */
+@com.raizlabs.android.dbflow.annotation.TypeConverter
+public class MilisecondsLocalDateAdapter extends TypeConverter<Long, LocalDate> {
```

Comment:
rename it to `LocalDateAdapter ` or `LocalDateDatabaseAdapter `. The milliseconds is not important here